sale_commission/models/sale_commission.py

```diff
@@ -23,6 +23,12 @@
 from openerp import models, fields, api, exceptions, _
 
 
+PERIOD_MONTH = "monthly"
+PERIOD_QUARTER = "quaterly"
+PERIOD_SEMI = "semi"
+PERIOD_YEAR = "annual"
```

Comment:
Is this use outside of the selection?
Wouldn't it be better to make a single constant that defines the whole selection list?